lib/belay_brokerage.ex

```diff
@@ -35,6 +36,15 @@ defmodule BelayBrokerage do
     end
   end
 
+  @spec update_investor(String.t(), String.t(), investor()) :: {:ok, Investor.t()} | {:error, Ecto.Changeset.t()}
```

Comment:
Update and upsert? Seems confusing on when to use which

Reply:
I'm going to change the upsert to a create, we never needed the concept of an upsert. I just wanted to do it separately so I could update all our refs to point to the ref without adding breaking changes.
I already got the changes that replace the upsert with a create, I'll add it next